src/components/Editor/Resources/ResourceListSearch.vue

```diff
@@ -21,48 +21,80 @@
   -->
 
 <template>
-	<Multiselect
-		class="resource-search"
-		:options="matches"
-		:searchable="true"
-		:internal-search="false"
-		:max-height="600"
-		:show-no-results="true"
-		:show-no-options="false"
-		:placeholder="placeholder"
-		:class="{ 'showContent': inputGiven, 'icon-loading': isLoading }"
-		open-direction="bottom"
-		track-by="email"
-		label="displayName"
-		@search-change="findResources"
-		@select="addResource">
-		<template #option="{ option }">
-			<div class="resource-search-list-item">
-				<Avatar
-					:disable-tooltip="true"
-					:display-name="option.displayName" />
-				<div class="resource-search-list-item__label resource-search-list-item__label--single-email">
-					<div>
-						{{ option.displayName }}
+	<div class="resource-search">
+		<div class="resource-search__filter">
+			<ResourceSeatingCapacity
+				:value="capacity"
```

Comment:
```suggestion
				:value.sync="capacity"
```

then you don't need the method. alternatively you could emit `input` and make this a `v-model`

Reply:
Neat! I applied this to the feature checkboxes too.